runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1049,8 +1049,8 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadBatchMatMul(const Operator *o
 
   if (op->custom_options() == nullptr)
   {
-    param.adj_x = false;
-    param.adj_y = false;
+    param.adj_x = op->builtin_options_as_BatchMatMulOptions()->adjoint_lhs();
```

Comment:
This makes problem if `CUSTOM(BatchMatMul)` have empty custom option.

Reply:
Thanks, I will fix it.